fractal-puppy/Engine
# Bug/boss phase3 player movement

Now player movement during boss 3rd stage works properly.
To test:
- check that the player navigates properly in boss 3rd stage
- check that player can attack boss in 3rd stage
